### ISSUE TITLE
Gracefully handle if no Gateways or MUXes provisioned yet

### ISSUE THREAD

**Issue by arudell** [microsoft/SdnDiagnostics#111]
**Describe the bug**
In newly deployed environments, there may not yet be gateways or muxes deployed yet. This causes issues when running Get-SdnGateway or Get-SdnLoadBalancerMUX where it generates warnings and errors.

```
"<REMOVED>","2/23/2022 3:07:36 PM","Get-SdnLoadBalancerMux","Warning"," is reporting provisioningState: "
"<REMOVED>","2/23/2022 3:07:36 PM","Get-SdnLoadBalancerMux","Error","System.Management.Automation.ParameterBindingValidationException: Cannot bind argument to parameter 'ResourceRef' because it is an empty string.
   at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
   at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
at Get-SdnLoadBalancerMux, C:\Program Files\WindowsPowerShell\Modules\SDndiagnostics\2.2202.215.82917\modules\NetworkController\public\Get-SdnLoadBalancerMux.ps1: line 41
at Get-SdnInfrastructureInfo, C:\Program Files\WindowsPowerShell\Modules\SDndiagnostics\2.2202.215.82917\modules\NetworkController\public\Get-SdnInfrastructureInfo.ps1: line 89
at <ScriptBlock>, <No file>: line 1"
```

**Expected behavior**
We should gracefully indicate no resources provisioned if call to API endpoint is successful but no resources returned.
